developer-extension/src/panel/hooks/useEvents.ts

```diff
@@ -115,17 +116,18 @@ function listenEventsFromRequests(callback: (events: StoredEvent[]) => void) {
   function beforeRequestHandler(request: chrome.devtools.network.Request) {
     const url = new URL(request.request.url)
 
-    const intake = /^(rum|logs).browser-intake-/.exec(url.hostname)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏This was used instead of the hardcoded list of all intake domains because when that was implemented, we chose to keep the staging domain private. Now that it is [part of the codebase](https://github.com/DataDog/browser-sdk/blob/323e0726f07a50c2154bde0863b86c366b9977c1/packages/core/src/domain/configuration/intakeSites.ts#L1), I guess this rule doesn't apply anymore. Let's add it to the `INTAKE_DOMAINS` to make the extension work on staging as well.